GameServerLib/Logic/Config.cs

```diff
@@ -19,16 +19,24 @@ public class Config
         public bool ChatCheatsEnabled { get; private set; }
         public bool MinionSpawnsEnabled { get; private set; }
 
-        public Config(string path)
+        public Config(string path, string inputJson = "")
         {
-            LoadConfig(path);
+            LoadConfig(path, inputJson);
         }
 
-        private void LoadConfig(string path)
+        private void LoadConfig(string path, string inputJSON)
         {
             Players = new Dictionary<string, PlayerConfig>();
 
-            var data = JObject.Parse(File.ReadAllText(path));
+            var data = (inputJSON == "") ? JObject.Parse(File.ReadAllText(path)) : JObject.Parse(inputJSON);
```

Comment:
This thing should probably be moved to the `GameServerApp`, and only the parsing result should be passed forward to the game server lib.